Documentation/boostrap-new-os.md

```diff
@@ -3,6 +3,13 @@
 When adding support for a new OS, both the native (C++) and managed (C#) code components in the coreclr, corefx and core-setup repos need to be compiled. Making the native components build on a new OS is a quite straightforward process and the tools like CMake, Clang C/C++ compiler, python and awk that are required for the build are available for almost all platforms. 
 But for the managed code compilation, we have a chicken and egg problem. A .NET CLI toolchain is needed to build managed components in the coreclr, corefx and core-setup repos, but it is not available for the target platform yet. 
 The way to solve this problem is to create a bootstrap CLI by taking an existing CLI as a "seed" and replacing native components in it by the native components that we build for the new platform. The seed CLI we use has to be for the same processor architecture as our new platform or we have to use a seed cli that we build from sources on another platform that is already supported and opt for not crossgenning the managed components. 
+## When is bootstrapping necessary?
+Bootstrapping is necessary when:
+- Your available version of the SDK cannot build your desired version of the SDK.
+- You have no version of the SDK available for your platform.
+- You are building the SDK for a non-x64 platform.
+
+In practice, moat maintainers will bootstrap once every major version (e.g. 6.0.100, 7.0.100, etc).  If you bootstrap these initial versions once you will generally be able to build 6.0.101 with 6.0.100, 6.0.102 with 6.0.101, 7.0.101 with 7.0.100, and so on.  The source-build team attempts to avoid requiring bootstrapping more than once per major version but sometimes this is unavoidable - often when we move from one feature band to another, or if a major language feature is added, or if there are issues discovered with a given version of the SDK after release.  These types of situations will be announced on the source-build [discussions page](https://github.com/dotnet/source-build/discussions).
```

Comment:
```suggestion
In practice, most maintainers will bootstrap once every major version (e.g. 6.0.100, 7.0.100, etc).  If you bootstrap these initial versions once you will generally be able to build 6.0.101 with 6.0.100, 6.0.102 with 6.0.101, 7.0.101 with 7.0.100, and so on.  The source-build team attempts to avoid requiring bootstrapping more than once per major version but sometimes this is unavoidable - often when we move from one feature band to another, or if a major language feature is added, or if there are issues discovered with a given version of the SDK after release.  These types of situations will be announced on the source-build [discussions page](https://github.com/dotnet/source-build/discussions).
```